packages/blockchain-wallet-v4-frontend/src/data/wallet/sagas.js

```diff
@@ -63,10 +63,20 @@ export default ({ coreSagas }) => {
     yield call(coreSagas.kvStore.walletCredentials.fetchMetadataWalletCredentials)
   }
 
+  const updateMnemonicBackup = function* () {
+    yield put(actions.core.wallet.updateMnemonicBackup())
+  }
+
+  const triggerMnemonicViewedAlert = function* () {
+    yield put(actions.core.wallet.triggerMnemonicViewedAlert())
+  }
+
```

Comment:
just call `core` directly from templates and remove all of this redux boilerplate